
Support comments in the test ignore file (see ../files/test.exclude)

--- Python-3.12.1/Lib/test/libregrtest/cmdline.py.orig
+++ Python-3.12.1/Lib/test/libregrtest/cmdline.py
@@ -193,6 +193,7 @@
         items = getattr(namespace, self.dest)
         with open(value, encoding='utf-8') as fp:
             for line in fp:
+                if line.startswith('#'): continue
                 items.append((line.strip(), self.const))
 
 
